api/krusty/multiplepatch_test.go

```diff
@@ -303,20 +301,15 @@ patchesStrategicMerge:
 	m = th.Run("overlay", th.MakeDefaultOptions())
 	th.AssertActualEqualsExpected(m, expected)
 
-	// Technique 4: "patches:" field, one patch file.  Fails.
+	// Technique 4: "patches:" field, one patch file.
```

Comment:
nit: I'm debating whether we should move this test into `plugin/builtinpatchtransformer/PatchTransformer_test.go`? 

The way I think about it, `multipatch_test.go` may be testing some special behavior of multiple applied patches not specific to the `PatchTransformer` code. Say, the tests could be checking some behavior defined in other libraries that we call in `PatchTransformer` code. But we now explicitly loop through all the strategic merge patches in `PatchTransformer`, so maybe this code would be better situated there.

Reply:
Looks like this test is checking the difference in behavior between `patches` and `patchesStrategicMerge`.
So I think this test can't check if we put this test `PatchTransformer_test.go` bud are you want to move this code?

Reply:
I agree with your opinion!
I add those tests!